### PR TITLE
Lookup Service

### PR DESCRIPTION
**Pull request by KFloodBCGov**
# Description

This PR includes the following proposed change(s):

Adds in a Efiling Lookup Service for Fee lookup (and other info)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Locally building it (no functions in the service yet).

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-lookup-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    <generatePackage>ca.bc.gov.ag.csows.lookups</generatePackage>
+                    <schemaIncludes>
+                        <schemaIncludes>LookupFacade.wsdl</schemaIncludes>
+                    </schemaIncludes>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (akroon3r) [inline src/backend/libs/efiling-lookup-client/pom.xml:72]: OCD: missing newline, its fine if this is auto generated

File: src/backend/libs/efiling-lookup-client/src/test/java/ca/bc/gov/open/jag/efilinglookupclient/EfilingLookupClientTest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("Efiling Lookup Client Test Suite")
+public class EfilingLookupClientTest {
+
+
+    @DisplayName("CASE 1:")
+    @Test
+    public void callLookupTest() {
+
```
Comment (akroon3r) [inline src/backend/libs/efiling-lookup-client/src/test/java/ca/bc/gov/open/jag/efilinglookupclient/EfilingLookupClientTest.java:14]: are we ok with merging an empty test suite?

Comment (KFloodBCGov) [reply src/backend/libs/efiling-lookup-client/src/test/java/ca/bc/gov/open/jag/efilinglookupclient/EfilingLookupClientTest.java:14]: Right now this is an empty service, no functions implemented.

File: src/backend/libs/efiling-lookup-client/pom.xml (hunk 1 above)
Comment (KFloodBCGov) [reply src/backend/libs/efiling-lookup-client/pom.xml]: Should be fixed

**Pull request merged**